### ISSUE TITLE
Space cli should ask whether to put the binary in path

### ISSUE THREAD

**Issue by sharadregoti** [spaceuptech/space-cloud#1071]
### The problem faced currently?
Most users access space-CLI binary by going to the downloaded path & then run commands. It would be nice if space-cli gives a promt of adding it to the path.
### How can we solve it?

>If you want this feature to be implemented, give it a thumbs up reaction, so that we can determine which features are important to you.
>👍

**Comment by sharadregoti** [comment]
@YourTechBud what should we do about this issue ?

**Comment by prakashmishra1598** [comment]
Rather than prompting the user, I think it would be nice to have a tiny installer script that does it. Something like a one-liner: `curl <url>/get_spacecloud.sh | sh` that could 
download and move the binary to /use/local/bin when executed. 

Also, the script could prompt the user if they want to add it to the path or just leave it in the current folder rather than implementing this logic in the binary. This approach just seems a bit cleaner to me. 

cc: @sharadregoti @YourTechBud

**Comment by YourTechBud** [comment]
I agree.

**Comment by sharadregoti** [comment]
yes, we should write a script for this. The script should also enable auto completion for cli
